app/views/shared/_working_submission.html.erb

```diff
@@ -0,0 +1,96 @@
+    <div class="d-flex justify-content-end">
+      <span class="mx-3"><%= link_to "Approve", set_status_manager_path(user_id: @user.id, status: "approved", submission_id: @submission.id), data: { turbo_method: "post" }, class: "btn btn-primary" %></span>
+      <span><%= link_to "Reject", set_status_manager_path(user_id: @user.id, status: "rejected", submission_id: @submission.id), data: { turbo_method: "post" }, class: "btn btn-danger" %></span>
+    </div>
+  <% end %>
+  <div class="row">
+    <div class="col-2">
+      <%= image_tag "site-logo.png" %>
+    </div>
+    <div class="col-10 d-flex align-items-center justify-content-start">
+      <h1>EMPLOYEE WORKING HOLIDAYS FORM</h1>
+    </div>
+  </div>
+
+  <div class="d-flex mt-5">
+    <div class="col-3">
+      <span>Employee Name :</span>
+    </div>
+    <div class="col-9">
+      <span><%= @user.name %></span>
+    </div>
+  </div>
+
+  <div class="d-flex mt-3">
+    <div class="col-3">
+      <span>Employee ID :</span>
+    </div>
+    <div class="col-9">
+      <span><%= @user.emp_id %></span>
+    </div>
+  </div>
+
+  <div class="d-flex mt-4">
+    <div class="col-6">
+      <div class="d-flex">
+        <div class="col-4">
+          <span>Department :</span></div>
+        <div class="col-8 form-header">
+          <span><%= @user.department %></span>
+        </div>
+      </div>
+    </div>
+    <div class="col-6">
+      <div class="d-flex">
+        <div class="col-4">
+          <span>Job Title :</span>
+        </div>
+        <div class="col-8">
+          <span class="form-header"><%= @user.role.titleize %></span>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="d-flex">
+    <div class="col-3">
+      <span>Reporting manager :</span>
+    </div>
+    <div class="col-9">
+      <span><%= @user.manager&.name %></span>
+    </div>
+  </div>
+
+  <table class="table table-bordered mt-4 form-table">
+    <thead>
+    <tr>
+      <% @form.data.keys.each do |key| %>
+        <% unless ["reporting_manager"].include? key %>
+          <th scope="col" class="bg-secondary"> <%= key.titleize %> </th>
+        <% end %>
+      <% end %>
+    </tr>
+    </thead>
+    <tbody id="working-table">
+    <% @submission.data.except('reporting_manager').each do |keys, values| %>
+      <tr>
+        <% values.each do |key, value| %>
+          <td><%= value %></td>
+        <% end %>
+      </tr>
+    <% end %>
+    </tbody>
+  </table>
+
+  <% if @active_role == "accountants" && @user.role != "accountant" %>
+    <div class="d-flex justify-content-end">
+      <h4> Total: <%= @user.total_generator([@submission]) %></h4>
+    </div>
+
+    <div class="d-flex justify-content-center">
+      <%= link_to 'Excel', user_submissions_accountants_path(format: :xlsx, user_id: params[:user_id], _type: params[:_type], form_id: params[:form_id], submission_id: @submission.id), class: "btn btn-primary" %>
+    </div>
+  <% end %>
+
+</div>
```

Comment:
this piece of code is also repeating again and again

Reply:
Its for design of form,